### PR TITLE
fix: 🐛 Expose non-proxied polkadot client

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
#### JIRA Link

https://polymath.atlassian.net/browse/DA-244

### CODE REVIEW

File: src/base/Context.ts [hunk 1]
```diff
@@ -435,6 +435,15 @@ export class Context {
     return identity;
   }
 
+  /**
+   * @hidden
+   *
+   * Retrieve the non-proxied Polkadot client
```
Comment (polymath-eric) [inline src/base/Context.ts:441]: (nit)

```suggestion
   *  Retrieve the polkadot.js promise client
```

Very nit, but if someone is getting the "regular" version they shouldn't have to worry about what a proxy version is.

I had to look at the code to figure out what a Proxy meant in this context. I first thought proxy server where the connection is going through some jump box or VPN.

**Review by polymath-eric — COMMENTED** [review]
LGTM. Small comment nit.

I am surprised sonar cloud gives a check mark at 0%. Is this because it knows there is no logic in the method?

I am not sure if we need the proxy even, but thats might be a larger discussion.

File: src/base/Context.ts (hunk 1 above)
Comment (prashantasdeveloper) [reply src/base/Context.ts]: That makes sense. I have changed the docs as suggested.

**Review by polymath-eric — APPROVED** [review]
LGTM

**Comment by prashantasdeveloper** [comment]
I was bit in dilemma whether to write a test case for a one liner with almost no logic. And the build also passed so I skipped it. Probably, for such logics sonar is also chill about it.

In all the procedures proxied client is used, which throws error if signing account is not present. That is why I didn't remove it.

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**